examples/vello_editor/src/main.rs

```diff
@@ -116,6 +122,31 @@ impl ApplicationHandler for SimpleVelloApp<'_> {
         event_loop.set_control_flow(ControlFlow::Wait);
     }
 
+    fn new_events(&mut self, event_loop: &ActiveEventLoop, cause: StartCause) {
+        let now: Instant = Instant::now();
+    
+        match cause {
+            StartCause::Init => {
+                let next_blink_time = now + Duration::from_secs_f32(0.5);
+                self.last_blink_time = now;
+                event_loop.set_control_flow(ControlFlow::WaitUntil(next_blink_time));
+            }
+            StartCause::ResumeTimeReached { .. } => {
+                self.editor.cursor_blink();
+                // redraw
+                self.last_drawn_generation = text::Generation::default();
+                
+                self.last_blink_time = now;
+                let next_blink_time = now + Duration::from_secs_f32(0.5);
+                event_loop.set_control_flow(ControlFlow::WaitUntil(next_blink_time));
+                
+                if let RenderState::Active(state) = &self.state {
+                    state.window.request_redraw();
+                }
+            }
+            _ => {}
+        }
```

Comment:
Needs to handle [`WaitCancelled`](<https://docs.rs/winit/latest/winit/event/enum.StartCause.html#variant.WaitCancelled>).

Reply:
Done.